### DISCUSSION TITLE
Add Fallback Strategie adapter

### DISCUSSION THREAD

**Discussion by Chrispycode** [olimorris/codecompanion.nvim#314] [Ideas]
I use the same neovim setup at work and private but I only use copilot at work so I have to always change the Strategie adapter.
Another case where this would be handy is when copilot is down or bad internet connections because you can use ollama without internet.

**Comment by olimorris** [comment]
You can quickly change adapters in the chat using `ga` or start a chat with an adapter via `:CodeCompanionChat Ollama`. The former is great because you can pass the context of your previous conversation over to the other adapter.

**Reply by Chrispycode** [reply]
Yes this feature is nice and I already use it, but the issue is that it does not change the inline adapter and resets when I open a new chat. 
Is there a option to change adapters for chat and inline on the fly (only for the current neovim session) without editing the neovim configuration.

**Reply by olimorris** [reply]
That's a great shout.

I'll add this to the roadmap for next week

**Reply by Chrispycode** [reply]
Thank you very much love the plugin :heart:

**Reply by olimorris** [reply]
This is now in `v8.4.0`. You can set the `vim.g.codecompanion_adapter` variable to the name of an adapter in the config.
